src/test/java/com/karankumar/bookproject/backend/service/BookServiceTest.java

```diff
@@ -246,4 +253,51 @@ private Book createBookAndSetAllAttributes() {
     void deleteBooks() {
         bookService.deleteAll();
     }
+    
+    @Test
+    void bookWithoutPublisherWithAuthorSaved() {
+    	SoftAssertions softly = new SoftAssertions();
+
+        // when
+    	Book book = new Book("Book without author", author, toRead);
```

Comment:
`"Book without author"` or `"Book without publisher"` ?

Reply:
i'll correct that